src/Core/Bootstrap/ConfigurationLoader.php

```diff
@@ -79,6 +74,30 @@ protected function loadConfigurationFiles(Application $app, RepositoryContract $
         }
     }
 
+    /**
+     * Get the path to the routes cache file.
+     *
+     * @return string
```

Comment:
Remove docblock return statement and add `:void` return statement to method signature.